crates/ide-assists/src/handlers/add_braces.rs

```diff
@@ -32,7 +32,7 @@ pub(crate) fn add_braces(acc: &mut Assists, ctx: &AssistContext<'_>) -> Option<(
     let (expr_type, expr) = get_replacement_node(ctx)?;
 
     acc.add(
-        AssistId("add_braces", AssistKind::RefactorRewrite),
+        AssistId("add_braces", AssistKind::RefactorRewrite, None),
```

Comment:
It would likely be nicer to have a constructor function for the `None` case here, something like
```suggestion
        assist_id("add_braces", AssistKind::RefactorRewrite),
```
or 
```suggestion
        AssistId::refactor_rewrite(add_braces"),
```
To keep the group one long but the common case short?

Reply:
I force-pushed a new version with the second option.  I chose it over the first one because it resulted in more concise code.  Thanks for the suggestion.